src/js/utils/RouterUtil.js

```diff
@@ -18,6 +18,28 @@ function findRedirect(queryString) {
 
 const RouterUtil = {
   /**
+   * Replace query in url path
+   * options = {
+   *    pathname: "http://localhost:4200/#/catalog/packages/arangodb3?version=1.0.0&_k=ed535h",
+   *    query: "version",
+   *    value: "test"
+   * }
+   *
+   * Output: http://localhost:4200/#/catalog/packages/arangodb3?version=test&_k=ed535h
+   *
+   * @param {Object} options object config
+   * @returns {String} string with new query value
+   */
+  replaceQueryInPathString(options) {
```

Comment:
There is also the case when a query string has no value. Example: `?key=` or `?key`. This is probably not an expected case (but is valid and can happen). Since this is a general util we may want to make this function more general to handle all kinds of keys, but just keys with values.

Reply:
This method was created to only replace the value when existent nothing else, but I'm open to add that if more people feel like that will happen.

I rather see a unit test confirming this was wanted.

Reply:
@ahoskins have we decided what to do here?

Reply:
Talked offline: we will add more unit tests as commented below, and leave this method as-is.